samples/client/petstore/go/go-petstore/model_enum_arrays.go

```diff
@@ -10,6 +10,22 @@
 package petstore
 
 type EnumArrays struct {
-	JustSymbol string `json:"just_symbol,omitempty"`
-	ArrayEnum []string `json:"array_enum,omitempty"`
+	JustSymbol EnumArraysJustSymbol `json:"just_symbol,omitempty"`
+	ArrayEnum EnumArraysArrayEnum `json:"array_enum,omitempty"`
```

Comment:
This isn't quite right, it needs to be
```go
ArrayEnum []EnumArraysArrayEnum```
on this line and
```go
type EnumArraysArrayEnum string```
one line 25 below

I'll work on a fix...

Reply:
Fixed